src/kixi/heimdall/util.clj

```diff
@@ -1,7 +1,12 @@
 (ns kixi.heimdall.util
   (:require [clj-time.format :as tf]
             [clj-time.core :as t]
-            [clojure.java.io :as io]))
+            [clojure.java.io :as io]
+            [clojure.edn :as edn]))
+
+(defn get-config
+  [f]
+  (edn/read-string (slurp f)))
```

Comment:
get-config could go in config namespace. 

db-now below could go into a time one (there will probably be others that will end up in there too)

replacer, underscore->hyphen and hyphen->underscore could go into a conformer or coerecer namespace